ports/nrf/mpconfigport.h

```diff
@@ -332,10 +349,25 @@ long unsigned int rng_generate_random_word(void);
 
 #define MP_STATE_PORT MP_STATE_VM
 
-#if MICROPY_HW_USB_CDC
-#include "device/usbd.h"
-#define MICROPY_HW_USBDEV_TASK_HOOK extern void tud_task(void); tud_task();
-#define MICROPY_EXCLUDE_SHARED_TINYUSB_USBD_CDC (1)
+#if MICROPY_HW_ENABLE_USBDEV
+#ifndef MICROPY_HW_USB_CDC
+#define MICROPY_HW_USB_CDC (1)
+#endif
+#endif
+
+#if MICROPY_HW_ENABLE_USBDEV
```

Comment:
Any reason why this extra #if is needed here, can't the config below be part of the block just above with the `MICROPY_HW_USB_CDC` config?

Reply:
Oh thanks, yes I'd moved the block above to that spot to consolidate the two, then forget the final cleanup.